bundle/src/main/java/com/adobe/acs/commons/indesign/dynamicdeckdynamo/servlets/impl/TriggerDeckDynamoServlet.java

```diff
@@ -1,4 +1,23 @@
-package com.adobe.acs.commons.indesign.dynamicdeckdynamo.servlets;
+/*
+ * ACS AEM Commons
+ *
+ * Copyright (C) 2020 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.adobe.acs.commons.indesign.dynamicdeckdynamo.servlets.impl;
```

Comment:
Servlet to an impl, since this is exposed via Sling anyhow; no one should be calling this class directly in Java code.